src/httpUtils.ts

```diff
@@ -9,11 +9,18 @@ function buildUrlEncodedRequest(request: TTokenRequest): string {
   return queryString
 }
 
-export async function postWithXForm(url: string, request: TTokenRequest): Promise<Response> {
+interface PostWithXFormParams {
+  url: string
+  request: TTokenRequest
+  credentials?: RequestCredentials
```

Comment:
Should not be optional here now, after changing the type in InternalConfig :slightly_smiling_face: